akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,6 +52,11 @@ trait JournalCapabilityFlags extends CapabilityFlags {
    */
   protected def supportsSerialization: CapabilityFlag
 
+  /**
+   * When `true` enables tests which check if the journal supports deletes
+   */
+  protected def supportsDeletes: CapabilityFlag
```

Comment:
Default it to true to not break source compat for existing plugins?

Reply:
The override to true is in the JournalSpec